packages/sources/src/components/form/models/form-group-select.model.ts

```diff
@@ -11,7 +11,7 @@ export interface FormGroupSelect<TemplateFnReturnType> extends FormGroupBase<Tem
 }
 
 export interface SelectOptionGroup {
-  label: string;
+  label?: string;
```

Comment:
Zonder `label` heeft een "groep" werkvorm in deze context weinig zin. Ik denk dat je een andere oplossing moet zoeken.

Reply:
Mee eens, ik draai het terug